orchestration/dap_orchestration/config/__init__.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from dagster import ResourceDefinition
+from dagster_utils.config import configurator_aimed_at_directory
+from dagster_utils.typing import DagsterObjectConfigSchema
+
+preconfigure_for_mode = configurator_aimed_at_directory(os.path.dirname(__file__))
+
+
+def preconfigure_resource_for_mode(
```

Comment:
⛏️ This feels like it should be extracted to `dagster-utils` if it doesn't already exists there. Can you ticket if that's the case?